spec/support/authorised_request_helper.rb

```diff
@@ -0,0 +1,9 @@
+module AuthorisedRequestHelper
+  def authorise_requests!
```

Comment:
I can't see anywhere that we are using these spec helper functions (at the moment it looks like we are only testing unauthorised requests). Do we need to create some sort of test for an authorised request?

Reply:
I think I had another branch with a 'fake' endpoint that tested this... maybe once we have actual endpoints present we can rebase and test this?